File: src/components/JSONHandler/src/DeleteInteractionChoiceSet.cpp
#include "JSONHandler/DeleteInteractionChoiceSet.h"
#include "JSONHandler/RPC2Marshaller.h"

using namespace RPC2Communication;

DeleteInteractionChoiceSet::DeleteInteractionChoiceSet()
:RPC2Request( RPC2Marshaller::METHOD_DELETEINTERACTIONCHOICESET_REQUEST )
{}
 
DeleteInteractionChoiceSet::~DeleteInteractionChoiceSet()
{}

void DeleteInteractionChoiceSet::setInteractionChoiceSetId(unsigned int s)
{
    interactionChoiceSetID = s;
}
        
unsigned int DeleteInteractionChoiceSet::getInteractionChoiceSetId() const
{
    return interactionChoiceSetID;
}

File: mobile/android/SyncProxyAndroid/src/com/ford/syncV4/proxy/rpc/DeleteInteractionChoiceSet.java
package com.ford.syncV4.proxy.rpc;

import java.util.Hashtable;

import com.ford.syncV4.proxy.RPCRequest;
import com.ford.syncV4.proxy.constants.Names;

public class DeleteInteractionChoiceSet extends RPCRequest {

    public DeleteInteractionChoiceSet() {
        super("DeleteInteractionChoiceSet");
    }
    public DeleteInteractionChoiceSet(Hashtable hash) {
        super(hash);
    }
    public Integer getInteractionChoiceSetID() {
        return (Integer) parameters.get( Names.interactionChoiceSetID );
    }
    public void setInteractionChoiceSetID( Integer interactionChoiceSetID ) {
        if (interactionChoiceSetID != null) {
            parameters.put(Names.interactionChoiceSetID, interactionChoiceSetID );
        }
    }
}